### ISSUE TITLE
Make it clear that only certain messengers support webxdc

### ISSUE THREAD

**Issue by WofWca** [webxdc/website#35]
![image](https://user-images.githubusercontent.com/39462442/218974469-ede78f71-e57c-4a07-a3fd-1fe2834bc60f.png)

Basically revert a3abc62e0f41fc43a13fa719076bb6b6dd06eaf0.

I think it adds confusion, e.g. some may assume that this technology works in all the messengers, they'll be puzzled by how it works. I know that they can figure it out after a bit of digging, but sooner is better, I think.

I think to better understand how to write things we need to answer here for whom the website is made primarily. Is it regular users, messenger developers, or app developers? I think it's the latter. So I'm for making it clear that only Delta Chat supports webxdc for now.

**Comment by r10s** [comment]
of course, there were some discussions about that before (as you know by the history :), but at the end, it was that. 

i also do not think, many ppl come to this page _not_ via delta chat.

target group of the page are app developers and maybe messenger developers. i think, both with easily get that this does _not_ just work eg. in whatsap - but it underlines the offering of webxdc, that is not bound to delta, i think, this is a very important bit.

at some point, we will probably revise the page, and at some point the target group may also change - but for now, i would leave it as is.

**Comment by hpk42** [comment]
If we need to discuss "other messengers" at all then we can rightfully 
say that we are in discussion with messenger developers ("quiet" and "briar") 
but that for now, DC is the one spearheading the webxdc concept.

**Comment by Simon-Laux** [comment]
Meanwhile https://cheogram.com has added support for webxdc, so maybe it's time to add some section for messengers that support webxdc, either on the websites or in the docs?

**Comment by hpk42** [comment]
sure, is there a good link to cheogram's state of webxdc support?

**Comment by r10s** [comment]
ftr, we meanwhile summed up things wrt other messengers at https://delta.chat/en/2023-06-28-webxdc-import-export#all-fine-and-dandy-but-what-about-other-messengers-supporting-webxdc

**Comment by Simon-Laux** [comment]
> ftr, we meanwhile summed up things wrt other messengers at https://delta.chat/en/2023-06-28-webxdc-import-export#all-fine-and-dandy-but-what-about-other-messengers-supporting-webxdc

not on the main page, you can not expect people visiting webxdc.org knowing the deltachat blog and looking for information there.

**Comment by r10s** [comment]
i just added the link as it may be helpful to solve this issue. i did not say that i expect expect people visiting webxdc.org  knowing the deltachat blog  🤷‍♂️

**Comment by hpk42** [comment]
the entry page and docs and apps page now all make it clear which messengers support webxdc -- see webxdc.org
